editing/edit-feature-attachments/src/main/java/com/esri/samples/edit_feature_attachments/EditFeatureAttachmentsSample.java

```diff
@@ -273,26 +260,25 @@ private void deleteAttachment(int attachmentIndex) {
   private void applyEdits(ServiceFeatureTable featureTable) {
 
     // apply the changes to the server
-    ListenableFuture<List<FeatureEditResult>> editResult = featureTable.applyEditsAsync();
-    editResult.addDoneListener(() -> {
-      try {
-        List<FeatureEditResult> edits = editResult.get();
-        // check if the server edit was successful
-        if (edits != null && edits.size() > 0) {
-          if (!edits.get(0).hasCompletedWithErrors()) {
-            displayMessage(null, "Edited feature successfully");
-          } else {
-            if (edits.get(0).getError() != null) {
-              throw edits.get(0).getError();
+    featureTable.applyEditsAsync().toCompletableFuture().whenComplete(
+      (edits, ex) -> {
+        if (ex == null) {
+          // check if the server edit was successful
+          if (edits != null && edits.size() > 0) {
+            if (!edits.get(0).hasCompletedWithErrors()) {
```

Comment:
The only methods that call `applyEdits` are attaching or deleting a single attachment and then immediately calling `applyEdits`, so the method never has to deal with more than one edit AFAIK. Method name is a bit unfortunate since it's really just "apply edit".

Reply:
I'm also noting feedback elsewhere about methods involving an async operation and not being noted as async in the method name. I'll add this to the issue to look more generally at applying edits in samples.